### PR TITLE
New MCT Layout Plugins

### PR DESCRIPTION
**Pull request by Duncan-McD**
# New MCT Layout Plugins

### Summary of each plugin
- Time Conductor - allows for more ease when looking at data in various chunks of time
- Plan/Timeline - lets you make plans or timelines for later events in the mission
- Notebook - allows for timestamped note taking during the mission
- LADTable - Displays the latest available data for a piece or pieces of telemetry
- Filters - Enables filtering of data tables

(also added images/pan.png logo to the mct/public folder for use on ground software layouts if we desire)

### CODE REVIEW

File: MCT/public/dual.html [hunk 1]
```diff
@@ -12,15 +12,22 @@
 
 <body>
     <script>
+
+        const THIRTY_SECONDS = 30 * 1000;
```
Comment (shihaocao) [inline MCT/public/dual.html:16]: are these millisceonds?

Comment (shihaocao) [reply MCT/public/dual.html:16]: 30 seconds of milliseconds

File: MCT/public/dual.html [hunk 2]
```diff
@@ -12,15 +12,22 @@
...
+        const TWO_HOURS = ONE_HOUR * 2;
+        const ONE_DAY = ONE_HOUR * 24;
+
         //indicates the location of the openmct libraries and assets
         openmct.setAssetPath('openmct');
         //installs plugins for creation and storage of user objects like layouts in your Local Browser Storage
         openmct.install(openmct.plugins.LocalStorage());
         openmct.install(openmct.plugins.MyItems());
         //installs new plugins for viewing telemetry and analyzing telemetry
-        openmct.install(openmct.plugins.AutoflowView({
```
Comment (shihaocao) [inline MCT/public/dual.html:21]: what was this and why delete

File: MCT/public/dual.html (hunk 1 above)
Comment (Duncan-McD) [reply MCT/public/dual.html:16]: yeah - MCT understands milliseconds for the timeline

File: MCT/public/dual.html (hunk 2 above)
Comment (Duncan-McD) [reply MCT/public/dual.html:21]: I replaced telemetry.fixed with telemetry.panel so moving plots is more flexible and the view is automatically a panel not fixed in place when added (it is what NASA OpenMCT recommends)

**Review by shihaocao — APPROVED** [review]
dub

**Pull request merged**